drv/lpc55-update-server/src/main.rs

```diff
@@ -883,6 +915,33 @@ fn copy_from_caboose_chunk(
     Ok(())
 }
 
+fn copy_from_flash_range(
```

Comment:
I don't love that this is basically a copy/paste of the caboose function but I was finding the different errors (`UpdateError`  vs `RawCabooseError`) to be a pain to handle. Maybe I should take a close look?

Reply:
You could call `map_err` ion the result if it's the same code other than error.

Reply:
Yeah I tried looking at this again and it also turns out that because the page is fixed size vs caboose this also doesn't quite work. I think the separate function is fine for now.

Reply:
I'm struggling with the same thing w/r/t plumbing through the Attest.log function. It's only slightly different than the Attest.cert and Update functions but different enough to make finding a common implementation difficult. We'll have plenty of opportunities to find an approach w/ less copy-pasta in the future.